tinygrad/tensor.py

```diff
@@ -503,6 +503,10 @@ def layernorm(self, axis=-1, eps:float=1e-5) -> Tensor:
     y = (self - self.mean(axis, keepdim=True))
     return y.mul((y*y).mean(axis, keepdim=True).add(eps).rsqrt())
 
+  def instancenorm(self, axis=-1, eps:float=1e-5) -> Tensor: 
+    m, y = self.mean(axis, keepdim=True), (self*self).mean(axis, keepdim=True)
+    return (self - m) / ((y-m*m) + eps).sqrt()
```

Comment:
Hmm, how is this different from layernorm?

Reply:
looks like they are the same, I've removed this method